src/api/controllers/bookmarkController.js

```diff
@@ -0,0 +1,74 @@
+import status from '../../helpers/constants/status.codes';
+import errorSender from '../../helpers/error.sender';
+import errorMessage from '../../helpers/constants/error.messages';
+import model from '../models/index';
+
+const { Bookmark } = model;
+/**
+ * @class
+ */
+export default class BookmarkController {
+  /**
+   * This functions stores the bookmarked articles
+   * If the user request this operation for thesecond time it will delete the boorkmark
+   * This is to create the toggle effect.
+   * @static
+   * @param {object} req the request sent to be excuted
+   * @param {object} res the response after execution
+   * @return {object} void
+   * @memberof BookmarkController
+   */
+  static async addBookmark(req, res) {
+    const user = req.user.user.username;
+    const { slug } = req.params;
+
+    try {
+      await Bookmark.create({
+        articleSlug: slug,
+        username: user
+      });
+
+      res.status(status.CREATED).json({
+        status: 201,
+        message: 'Bookmarked successfully'
+      });
+    } catch (e) {
+      await Bookmark.destroy({
+        where: {
+          username: user,
+          articleSlug: slug
+        }
+      });
+      res.status(status.OK).json({
+        status: 200,
+        message: 'Successfully removed bookmark'
+      });
+    }
+  }
+
+  /**
+ * This funcrtion gets the users  bookmarks from the database.
+  * @static
+   * @param {object} req the request sent to be excuted
+   * @param {object} res the response after execution
+   * @return {object} void
+   * @memberof BookmarkController
+ */
+  static async getBookmark(req, res) {
+    const user = req.user.user.username;
+    const emptyBookmarkList = 0;
+    const bookmarkList = await Bookmark.findAll({
+      where: {
+        username: user,
+      }
```

Comment:
can you include some article's details?? because I think the user will need to see something like article's title, ...